src/templates/hashlists/detail.template.html

```diff
@@ -20,6 +20,10 @@ <h2>Hashlist details</h2>
         <td>Member of Access Group:</td>
         <td>[[accessGroup.getGroupName()]]</td>
       </tr>
+      <tr>
+        <td>Creator:</td>
```

Comment:
Suggestion: make this also: 'created by' to make it consistent with the hashlist overview

Reply:
Fixed in https://github.com/hashtopolis/server/pull/778/commits/5892721e6e9f5b10efee6a3683f1f99194c8a106